cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,71 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignUpPage from '../support/pages/signUp.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import faker from 'faker';
 
 describe('Sign Up page', () => {
+  let name;
+  let email;
+  let password;
+  const signUpPage = new SignUpPage();
+
   before(() => {
+    cy.task('db:clear');
+  });
+
+  beforeEach(() => {
+    cy.visit('/#/register');
+    name = faker.name.firstName();
+    email = faker.internet.email();
+    password = faker.internet.password();
+  });
+
+  it('should create a new user', () => {
+    signUpPage.fillName(name);
+    signUpPage.fillEmail(email);
+    signUpPage.fillPassword(password);
+
+    signUpPage.submitSignUpForm();
+
+    cy.contains('Welcome!');
```

Comment:
It's more reliable to use data-qa attributes instead of text content for selecting elements, as these attributes are less likely to change. If the 'Welcome!' message changes in the future, this test will break.

Reply:
I made variables for all messages, it will improve maintaining of all tests in the future. They are stored at alertMessages.js
I have problems with data-qa attributes on my local machine
https://mate-parttime.slack.com/archives/C02J1F3EUDB/p1691237811581409

Reply:
I made variables for all messages, it will improve maintaining of all tests in the future. They are stored at alertMessages.js
I have problems with data-qa attributes on my local machine
https://mate-parttime.slack.com/archives/C02J1F3EUDB/p1691237811581409

Reply:
I have special file with all message texts, it will be easy to maintain.